lxd/operations.go

```diff
@@ -613,7 +627,13 @@ func operationsGet(d *Daemon, r *http.Request) response.Response {
 		}
 
 		// Get operation data.
-		ops, err := client.UseProject(projectName).GetOperations()
+		var ops []api.Operation
+		switch allProjects {
+		case true:
+			ops, err = client.GetOperations()
```

Comment:
Perhaps im misunderstanding this, but looking at how `operationsGet()` API handler works, if there is no project included in the request (it isnt here because client.UseProject()` hasn't been used) then it will default to the `default` project.

I would expect to see new client functions added for `GetOperations*AllProjects()` like we did for `GetInstanc*AllProjects`.

Reply:
Ok I understand it now. I neglect how the old way works before 😅. Will fix it too.